_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -60,8 +60,16 @@ The `title` elements of embedded documents, such as those in `iframe`, `object`,
 
 The [HTML specification - The `title` element](https://html.spec.whatwg.org/#the-title-element) requires that a document only has one `title` element, and that it is a child of the `head` element of a document. However, current HTML specification also describes what should happen in case of multiple titles, and titles outside the `head` element. Because of this, neither of these validation issues causes a conformance problem for WCAG.
 
+### Related rules
+
 - [HTML page has non-empty title](https://act-rules.github.io/rules/2779a5)
+
+### Bibliography
+
 - [Understanding Success Criterion 2.4.2: Page titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
+
+### Assumptions
```

Comment:
Not sure where this came from, but I don't think we have an Assumptions sub heading in the Background. 
We could keep the links to G88 and H25 in the Bibliography, and move the last sentence ("A prerequisite...") to the main background section.

Reply:
This was from a change request from @Jym77.

What is the consensus here?

Reply:
https://github.com/act-rules/act-rules.github.io/pull/1729#discussion_r742777198
Should be under the (existing) `## Assumptions` heading, not under a new `## Background > ### Assumptions` one.

Reply:
I agree with @Jym77's suggestion. Move the assumptions to the existing heading, and keep the links to the related techniques under the bibliography heading.

Reply:
Updated